docs/developer/architecture/logs.rst

```diff
@@ -0,0 +1,120 @@
+Log Centralisation
+==================
+
+Context
+-------
+
+MetalK8s value is to provide, out of the box, some services in order to ease
+the monitoring and operation of the platform as well as workload running on top
+of it.
+
+MetalK8s provides monitoring service with Prometheus and AlertManager in
+order to expose MetalK8s and workloads metrics and alerts.
+
+The logs generated by the platform and the workloads constitute an essential
+piece of information that can be very helpful when it comes to understand the
+root cause of a failure or a performance degradation. Because of the
+distributed nature of MetalK8s and workloads running of top of it, it is
+important to have some tooling to ease the analysis of near real time and past
+logs from a central endpoint. As such those logs should be stored on the system
+for a configurable period of time. The logs analysis functionalities should be
+accessible through an API and a UI. It is also important to ease correlation
+with other type of indicators such as KPIs and alerts.
```

Comment:
`s/type/types`

The sentence is unclear or incomplete? + using KPIs as an example indicator feel incorrect or something...

Reply:
When we talk about accessibility through a UI, are we talking about the Grafana UI or MetalK8s UI (can be achieved in the MetalK8s UI by consuming Loki's HTTP API as documented here: https://github.com/grafana/loki/blob/master/docs/api.md#lokis-http-api)

Reply:
updated